.github/workflows/test-partner-driver.yml

```diff
@@ -0,0 +1,50 @@
+name: Test All Partner Drivers
+
+on:
+  schedule:
+    - cron: "0 * * * *"
+  push:
```

Comment:
maybe on push to `main`?

Reply:
I think all branches is actually correct, since PRs would affect the runners and should ensure everything is working (even if not passing)